doc/JFLA2022/paper.tex

```diff
@@ -59,6 +59,8 @@
 \newcommand{\texniccenter}{{\TeX}nicCenter}
 \newcommand{\makefile}{\texttt{Makefile}}
 \newcommand{\latexeditor}{LEd}
+\newcommand{\Hydras}{Hydras \& Co.\xspace}
+\newcommand{\Coqcommunity}{\emph{Coq-community\xspace} }

```

Comment:
```suggestion
\newcommand{\Coqcommunity}{Coq-community\xspace}
```
I think we want to put the `\emph{...}` where we actually use the macro, for more flexibility. Do we really want Coq-community emphasized everywhere? I think this is what is causing the issues with `~\cite{XY}`.

Reply:
I agree with with your suggested change.  If we want to emphasize for instance the first occurrence of Coq-community, in the paper, it is still possible to write `\emph{\Coqcommunity}` . Is it usual ?

Reply:
Yes, I think writing `\emph{\Coqcommunity}` is the standard way to do it.